### PR TITLE
Optional Alert Message Field

### LINKED ISSUE #132: Support For Arbitrary Alert Messages
Heuristic configurations should be able to define some `metadata` that holds arbitrary user text. As of now, alerts just generate heuristic specific data fields that may not provide the necessary context for on-call individuals/analysts that work to triage and respond to an alert.

### PR DESCRIPTION
**Pull request by epociask**
## Fixes Issue

Fixes #132 
## Changes proposed
* Added an optional `Message` field to heuristic configurations that allow for arbitrary text to be enclosed within an alert

### CODE REVIEW

File: e2e/e2e_test.go [hunk 1]
```diff
@@ -376,14 +395,20 @@ func Test_Fault_Detector(t *testing.T) {
 	reader, err := bindings.NewL2OutputOracleCaller(predeploys.DevL2OutputOracleAddr, l1Client)
 	assert.Nil(t, err)
 
+	// Assign alert message
+
+	alertMsg := "the fault, dear Brutus, is not in our stars, but in ourselves"
```
Comment (adrain-cb) [inline e2e/e2e_test.go:400]: Nice

File: internal/alert/interpolator.go [hunk 2]
```diff
@@ -21,12 +21,16 @@ const (
...
+	*Message:*
+	%s
+
 	`
 )
 
 // Interpolator ... Interface for interpolating messages
 type Interpolator interface {
-	InterpolateSlackMessage(sUUID core.SUUID, message string) string
+	InterpolateSlackMessage(sUUID core.SUUID, assess string, msg string) string
```
Comment (adrain-cb) [inline internal/alert/interpolator.go:33]:
```suggestion
	InterpolateSlackMessage(sUUID core.SUUID, content string, msg string) string
```

File: internal/core/constants.go [hunk 3]
```diff
@@ -133,6 +133,22 @@ func StringToHeuristicType(stringType string) HeuristicType {
 	}
 }
 
+// AlertPolicy ... The alerting policy for a heuristic session
+type AlertPolicy struct {
+	Dest string `json:"destination"`
```
Comment (adrain-cb) [inline internal/core/constants.go:138]: Is an alert policy string the behavior we want here? What if someone wants to alert multiple sources with one heuristic? This seems limiting

Comment (epociask) [reply internal/core/constants.go:138]: https://github.com/base-org/pessimism/issues/81

**Review by adrain-cb — APPROVED** [review]
Lgtm

**Pull request merged**